basic structure from my tasks

Serhii-Fesianov opened the pull request.

Serhii-Fesianov committed:
basic structure from my tasks

ordila commented on src/components/BurgerComponent/BurgerComponent.jsx:
import s from "./BurgerComponent.module.css"; 

У цьому проекті ми домовились , що імпортуємо стилі за допомогою змінної з назвою 'styles'
Приклад : 
import **styles** from "./BurgerComponent.module.css";

ordila commented on src/components/BurgerComponent/BurgerComponent.module.css:
.svgBurger {
  stroke-width: 3px;
  stroke: #fafafa;
}

stroke: **#fafafa;**  --> потрібно використовувати css variables for colors (you can find them in index.css)

ordila commented on src/components/BurgerMenuModal/BurgerMenuModal.jsx:
Така ж сама ситуація з назвою змінної для стилів , виправиш всюди.

ordila commented on src/components/BurgerMenuModal/BurgerMenuModal.module.css:
Змінні для кольорів

Serhii-Fesianov committed:
fix styles and adaptive 375/768

Header Component
UserSetsModal(basic)
SureLogOutModal
BurgerComponentModal(without userBar)

ordila committed:
Merge branch 'main' into header-feature

ordila committed:
fix inde.js component
